### PR TITLE
Added custom scrollbar

### PR DESCRIPTION
**Pull request by abhinav-m22**
## Related Issue

Added a custom scrollbar to the webpage to improve the overall design.

* Closes: #391 

### Describe the changes you've made

The basic browser scrollbar is changed to a custom theme-based scrollbar to improve the UI/UX of the web page.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

I have setup the website on my local environment and then run it and tested the changes.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| image | image|

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by Mustafiz04 — APPROVED** [review]
The scroll bar looks nice to me. Color, gradient, and all in both dark and light modes.

**Review by Anmol-Baranwal — COMMENTED** [review]
I have checked, and the code works on most modern browsers. However, it's always a good practice to write code that ensures cross-browser compatibility, as it helps maintain consistent rendering and functionality across different browsers.

I have attached the sample code for your reference. Please review it and let me know your thoughts. Additionally, I recommend testing the code on the respective browsers to ensure it works as expected.

```css
/* Firefox */
* {
  scrollbar-width: thin;
  scrollbar-color: #8034e4 #E638EAFF;
}

/* Internet Explorer and Microsoft Edge */
*::-ms-scrollbar {
  width: 16px;
}

*::-ms-scrollbar-track {
  background: transparent;
}

*::-ms-scrollbar-thumb {
  background: linear-gradient(#8034e4, #E638EAFF);
}
```

**Review by vinzvinci — COMMENTED** [review]
Thanks for contributions! Please other mentor and developers guild to review this PR!

**Comment by mkubdev** [comment]
> I have checked, and the code works on most modern browsers. However, it's always a good practice to write code that ensures cross-browser compatibility, as it helps maintain consistent rendering and functionality across different browsers.
> 
> I have attached the sample code for your reference. Please review it and let me know your thoughts. Additionally, I recommend testing the code on the respective browsers to ensure it works as expected.
> 
> ```css
> /* Firefox */
> * {
>   scrollbar-width: thin;
>   scrollbar-color: #8034e4 #E638EAFF;
> }
> 
> /* Internet Explorer and Microsoft Edge */
> *::-ms-scrollbar {
>   width: 16px;
> }
> 
> *::-ms-scrollbar-track {
>   background: transparent;
> }
> 
> *::-ms-scrollbar-thumb {
>   background: linear-gradient(#8034e4, #E638EAFF);
> }
> ```

Hello! This is a great advice! 💯

**Comment by mkubdev** [comment]
Hello @abhinav-m22 :wave: Can you add @Anmol-Baranwal suggestion so that we merge your PR ?

**Comment by abhinav-m22** [comment]
> Hey @abhinav-m22, do you need help? (:

Actually I was a bit inactive since last few days so I couldn't reply... I'll implement the changes till tomorrow and let you know for any help
Thanks!!

**Comment by abhinav-m22** [comment]
Hey guys @mkubdev @Mustafiz04 @vinzvinci @Anmol-Baranwal I've made the required changes!
Please have a look now

File: styles/globals.css [hunk 1]
```diff
@@ -179,3 +179,40 @@
     }
   }  
 }
+
+/*Scroll Bar*/
```
Comment (krishstwt) [inline styles/globals.css:183]: This should be inside the layer of base shouldn't it be?

**Review by krishstwt — CHANGES_REQUESTED** [review]

**Comment by abhinav-m22** [comment]
Please check now

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
There are two instances where `@layer base` is defined, and one can be removed to eliminate unnecessary duplicate code. 
To address this issue, please merge the two blocks.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me. Thanks for the changes.

**Review by vinzvinci — APPROVED** [review]
LGTM!

**Review by krishstwt — APPROVED** [review]

**Pull request merged**